extension-manifest-v3/package.json

```diff
@@ -13,7 +13,7 @@
     "dev": "npm run build.chromium -- --watch",
     "dev.safari": "npm run build.safari -- --watch",
     "release": "web-ext build",
-    "start": "web-ext run --target=chromium",
+    "start": "web-ext run --target=chromium --no-reload",
```

Comment:
why `--no-reload` ?

Reply:
Because it works worse when turned on ;) For example, if it is on, it forces closing an open tab with a page from the extension, and dev-tools for the background process do not reload. It is better to keep it that way, as the extension reloads when files are changed. You can reload the page, and see the difference. As we can have HMR, the DX is much better that way.